nestkernel/nest_time.cpp

```diff
@@ -77,7 +78,7 @@ Time::compute_max()
   const tic_t tmax = std::numeric_limits< tic_t >::max();
 
   tic_t tics;
-  if ( lmax < tmax / Range::TICS_PER_STEP ) // step size is limiting factor
+  if ( lmax < tmax * Range::TICS_PER_STEP_INV ) // step size is limiting factor
```

Comment:
And we probably don't need to bother with changing these, or inside limit, or inside set_resolution and so on... just in the code that gets deeply looped, which is Time constructors and operators on Time objects.